velox/exec/OutputBuffer.cpp

```diff
@@ -63,6 +63,29 @@ std::string ArbitraryBuffer::toString() const {
       hasNoMoreData());
 }
 
+void BufferStats::recordAddPage(const SerializedPage& data) {
+  auto numRows = data.numRows();
+  VELOX_CHECK(numRows, "SerializedPage's numRows must be valid");
```

Comment:
VELOX_CHECK(numRows.has_value()...)

Reply:
`has_value()` is indeed more descriptive. refactored.